test/AspNet.Security.OAuth.Providers.Tests/Discord/DiscordTests.cs

```diff
@@ -46,5 +48,76 @@ public async Task Can_Sign_In_Using_Discord(string claimType, string claimValue)
             // Assert
             AssertClaim(claims, claimType, claimValue);
         }
+
+        [Fact]
+        public async Task Authorization_Endpoint_Uri_by_Default_Does_Not_Contain_Prompt()
+        {
+            static void ConfigureServices(IServiceCollection services)
```

Comment:
For consistency, please add _Arrange, Act, Assert_ comments to the tests as appropriate.

Reply:
My bad, I'll add those back.  Thanks